docs/client_install/src/asciidoc/index.adoc

```diff
@@ -21,7 +21,7 @@
 * @@@ END COPYRIGHT @@@ 
 ////
 
-= Trafodion Client Installation Guide
+= Client Installation Guide
```

Comment:
Just relating this back to what I see on the web site. If I click here: http://trafodion.apache.org/docs/2.0.0/client_install/index.html, I see a title, "Trafodion Client Installation Guide". Is the change above going to change this to "Client Installation Guide"? And if so, why are we removing the name "Trafodion" from it?